Supabase/src/commonMain/kotlin/io/github/jan/supabase/Utils.kt

```diff
@@ -61,7 +61,7 @@ suspend inline fun <reified T> HttpResponse.bodyOrNull(): T? {
     return try {
         val text = bodyAsText()
         supabaseJson.decodeFromString<T>(text)
-    } catch(e: Exception) {
+    } catch(_: Exception) {
```

Comment:
Ensuring the coroutine is active here, causes `BucketApiTest.testExistsWithNonExistingFile` to fail. I think there's a deeper issue in that test that gets exposed. The test shouldn't still be running if the job is completed. I think there might be a missing `await` somewhere there.

Reply:
Then I'm wondering why only that test fails :thinking:

Reply:
Okay there seems to be a problem with this method in general, this also fails with ensure active:
```kotlin
val client = HttpClient()
val res = client.get("https://google.de")
println(res.bodyOrNull<String>())
```

Reply:
I'll address this in a separate PR